client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -0,0 +1,246 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { Button } from '@automattic/components';
+import { isWithinBreakpoint } from '@automattic/viewport';
+import { getQueryArg, removeQueryArgs, addQueryArgs } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import { useContext, useEffect, useState, useMemo } from 'react';
+import DocumentHead from 'calypso/components/data/document-head';
+import Notice from 'calypso/components/notice';
+import SidebarNavigation from 'calypso/components/sidebar-navigation';
+import useFetchDashboardSites from 'calypso/data/agency-dashboard/use-fetch-dashboard-sites';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { resetSite } from 'calypso/state/jetpack-agency-dashboard/actions';
+import {
+	checkIfJetpackSiteGotDisconnected,
+	getSelectedLicenses,
+	getSelectedLicensesSiteId,
+	getSelectedSiteLicenses,
+} from 'calypso/state/jetpack-agency-dashboard/selectors';
+import { errorNotice } from 'calypso/state/notices/actions';
+import { getIsPartnerOAuthTokenLoaded } from 'calypso/state/partner-portal/partner/selectors';
+import { serializeQueryStringProducts } from '../../partner-portal/lib/querystring-products';
+import SitesOverviewContext from './context';
+import DashboardBanners from './dashboard-banners';
+import useQueryProvisioningBlogIds from './hooks/use-query-provisioning-blog-ids';
+import { DASHBOARD_PRODUCT_SLUGS_BY_TYPE } from './lib/constants';
+import SiteAddLicenseNotification from './site-add-license-notification';
+import SiteContentHeader from './site-content-header';
+import SiteNotifications from './site-notifications';
+import SiteTopHeaderButtons from './site-top-header-buttons';
+import SitesDataViews from './sites-dataviews';
+
+import './style.scss';
+
+const QUERY_PARAM_PROVISIONING = 'provisioning';
+
+export default function SitesDashboardV2() {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const jetpackSiteDisconnected = useSelector( checkIfJetpackSiteGotDisconnected );
+	const isPartnerOAuthTokenLoaded = useSelector( getIsPartnerOAuthTokenLoaded );
+
+	const selectedLicenses = useSelector( getSelectedLicenses );
+	const selectedSiteLicenses = useSelector( getSelectedSiteLicenses );
+	const selectedLicensesSiteId = useSelector( getSelectedLicensesSiteId );
+
+	const isStreamlinedPurchasesEnabled = isEnabled( 'jetpack/streamline-license-purchases' );
+
+	const selectedLicensesCount = isStreamlinedPurchasesEnabled
+		? selectedSiteLicenses.reduce( ( acc, { products } ) => acc + products.length, 0 )
+		: selectedLicenses?.length;
+
+	const {
+		search,
+		currentPage,
+		filter,
+		sort,
+		// selectedSites,
```

Comment:
Is this staying here commented out as a reminder?

Reply:
Yep. I am still determining how we will handle selected sites now. That comes from the old dashboard, so as a reminder that we have that data/code there.

Reply:
Wouldn't it be better to add a comment to that line indicating that?

Reply:
It should be removed when we implement the bulk selection. I'll add a to-do comment.

Reply:
https://github.com/Automattic/wp-calypso/pull/87250/commits/bd5a383a45581553f04532907ec8d0eda49d252e